src/editors.cpp

```diff
@@ -396,10 +396,18 @@ void Editors::tabBarContextMenu(const QPoint &point)
 	if (!editorUnderCursor) act->setEnabled(false);
 	connect(act, SIGNAL(triggered()), SLOT(moveToOtherTabGroup()));
 
-	act = menu.addAction(tr("Move all to other view"));
-	act->setData(QVariant::fromValue<TxsTabWidget *>(tabGroup));
-	if (!editorUnderCursor) act->setEnabled(false);
-	connect(act, SIGNAL(triggered()), SLOT(moveAllToOtherTabGroup()));
+	if (tabGroups[0] == tabWidgetFromEditor(editorUnderCursor)) {
+		act = menu.addAction(tr("Move all others to other view"));
+		act->setData(QVariant::fromValue<LatexEditorView *>(editorUnderCursor));
+		if (!editorUnderCursor) act->setEnabled(false);

```

Comment:
this is dead code (never true)
I would suggest to handle the whole part (move ...) only if editorUnderCursor != nullptr

Reply:
whole part-> ll. 399-410

Reply:
So you don't want to have disabled entries that I see here?
![grafik](https://user-images.githubusercontent.com/102688820/204150637-a2b23686-d986-441e-82b7-8a83d306401a.png)

Reply:
the logic removes all the other entries , see some lines below, so I would stick with that logic.

Reply:
![grafik](https://user-images.githubusercontent.com/102688820/204152780-f1b25231-f1ac-4c3c-bbee-d3e553b795c7.png)